### ISSUE TITLE
Include playbackposition

### ISSUE THREAD

**Issue by thisisawesome1994** [bbilly1/tubearchivist#125]
Hi,

It would be Nice to include the saving of the playback position.

Awaiting Your response,

Joannes Wyckmans

**Comment by bbilly1** [comment]
Yes, I have considered that before. As of now, the media player is just the default html5 `<video>` tag, so far there are no other JS controls like playback position and therefore also no resume from timestamp, and other things like that. This will require quite a more complicated video player that there is now.

Honestly I'm not sure how useful this would be, I remember reading somewhere that the average video length on youtube is something like 10-11 minutes, so resume from previous position becomes less important than for something like Plex or Emby, where we talk about feature length movies or tv shows.

**Comment by thisisawesome1994** [comment]
Oh, but the video's I watch are between 5 and 85 minutes, were most are around 30-45 minutes. So for me it is usefull

**Comment by PiotrKrzyzek** [comment]
Another vote for this feature from me. I mostly watch longer videos such as D&D sessions ... thus a playback item would be neat.

Maybe we can wrap this request into a larger discussion about replacing the standard <video> tag with a more feature-full player?

**Comment by thisisawesome1994** [comment]
I dont know how that works. But if you make one, I Will surely join.

BTW, I think most people who are really into this (like terabytes big)  have a longer attention span then 5-10 minute cat video's.

**Comment by n8detar** [comment]
JS should be able to pull or set the video position with the current video player with the currentTime property. I used it with the cast integration. We can also use the ontimeupdate event. This could also be used to set a video as watched if say the progress gets to 90%.
```
var video = document.getElementById("video-item"); // Get the video
video.duration; // Get length of video
video.currentTime; // Get the position
video.currentTime = 5; // Set the position

video.addEventListener("timeupdate", myScript); // ontimeupdate Event
<video ontimeupdate="myScript"> // Or used in the video tag
```

**Comment by bbilly1** [comment]
This is now implemented in v0.1.2. please update!
